### DISCUSSION TITLE
Beam and Godot = 🖤  ?

### DISCUSSION THREAD

**Discussion by Leinnan** [beamable/BeamableProduct#3084] [Show and tell]
Some experiments with Beamable and Godot. I need to update it to latest versions, both Beamable and Godot.
https://github.com/Leinnan/godotBeamableTests
What do you think?

**Comment by cdhanna** [comment]
Is the `UnitedGodot` folder actually needed for the Beamable.Common package to compile?

**Reply by Leinnan** [reply]
AFAIK no, it is needed for the `Source/Runtime.Core` to compile.
